tst/testinstall/recordname.tst

```diff
@@ -28,6 +36,28 @@ gap> r.(str1023) := 4;
 Error, Record names must consist of less than 1023 characters
 gap> r.(str1024) := 5;
 Error, Record names must consist of less than 1023 characters
+gap> names := RecNames(r);;
+gap> SortedList(names) = [str1021, str1022a, str1022b, "x", "y"];
+true
+gap> RecNames( () );
+Error, no method found! For debugging hints type ?Recovery from NoMethodFound
+Error, no 1st choice method found for `RecNames' on 1 arguments
+gap> REC_NAMES( () );
+Error, RecNames: <rec> must be a record (not a permutation (small))
+gap> REC_NAMES_COMOBJ( () );
+Error, RecNames: <rec> must be a component object (not a permutation (small))
+gap> EQ_PREC( rec(), () );
+false
+gap> EQ_PREC( (), rec() );
+false
+gap> LT_PREC( rec(), () );
+false
+gap> LT_PREC( (), rec() );
+true
```

Comment:
The ordering of records with other records varies each time GAP is started, so tests would have to be carefully written (probably testing things like `A<B and B<C implies A<C`, or just cover the cases where the ordering is well-defined. Was too lazy to do that.